src/frontend/web_application/src/scenes/DiscussionList/components/ParticipantsIcon/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+import ParticipantIconLetter from '../../../../components/ParticipantIconLetter';
+import Icon from '../../../../components/Icon';
+import './style.scss';
+
+const ParticipantsIcon = ({ discussion }) => {
```

Comment:
`PureComponent` won't be of much help here, the contrary in fact, because it's "magic" and shallow-compares `props` and `nextProps`, not just `discussion.participants.length` which is the trigger here